templates/experiments/single_experiment_home.html

```diff
@@ -212,54 +212,215 @@ <h3 class="font-bold text-lg">Parent Experiments:
+                  </button>
+
+                  <button
+                    x-show="getActiveFilterCount() > 0"
+                    @click="clearAllFilters()"
+                    class="p-2 text-gray-400 hover:text-gray-600 h-10 w-10 flex items-center justify-center"
+                    title="Clear all filters">
+                    <span class="text-lg font-bold">×</span>
+                  </button>
+                </div>
+
+              <!-- Filters Panel -->
+                <div
+                  x-show="filterData.showFilters"
+                  class="absolute left-0 mt-2 p-4 bg-white border border-gray-200 rounded-lg shadow-lg z-10"
+                  style="width: fit-content;"
+                >
+                  <div class="space-y-3">
+                    <template x-for="(filter, index) in filterData.filters" :key="index">
+                      <div class="flex items-center gap-3">
+                      <!-- Where/AND Label -->
+                        <div class="text-gray-500 font-medium w-16">
+                          <span x-text="index === 0 ? 'Where' : 'AND'"></span>
+                        </div>
+                      <!-- Column Selector -->
+                        <div class="relative">
+                          <select
+                            x-model="filter.column"
+                            @change="updateOperatorsByColumn(index)"
+                            class="block w-40 py-2 px-3 border border-gray-200 rounded-lg bg-white text-sm"
+                          >
+                            <option value="">Column</option>
+                            <template x-for="(config, column) in filterData.columns" :key="column">
+                              <option :value="column" x-text="column"></option>
+                            </template>
+                          </select>
+                        </div>
+                      <!-- Operator Selector -->
+                        <div x-show="filter.column" class="relative">
+                          <select
+                            x-model="filter.operator"
+                            @change="triggerFilterChange()"
+                            class="block w-32 py-2 px-3 border border-gray-200 rounded-lg bg-white text-sm"
+                          >
+                            <template x-for="operator in filter.availableOperators" :key="operator">
+                              <option :value="operator" x-text="operator"></option>
+                            </template>
+                          </select>
+                        </div>
+                        <template x-if="filter.operator">
+                          <div class="relative">
+                          <!-- Multi-select dropdown-->
+                            <template x-if="['tags', 'versions'].includes(filter.column) && ['any of', 'all of'].includes(filter.operator)">
```

Comment:
can this use the column type or is it particularly specific to these two fields?
```suggestion
                            <template x-if="filterData.columns[filter.column]?.type === 'choice' && ['any of', 'all of'].includes(filter.operator)">
```